content/master/concepts/managed-resources.md

```diff
@@ -208,40 +208,142 @@ resource object is deleted from Kubernetes and the `deletionPolicy` is
+deleted. In other cases, non default `managementPolicies` take precedence over
+the `deletionPolicy` field. Keep in mind that this behavior is only applicable
+if the management policy alpha feature is enabled. To sum it up in a table:
+
+{{< table >}}
+| managementPolicies          | deletionPolicy   | result  |
+|-----------------------------|------------------|---------|
+| "*" (default)               | Delete (default) | Delete  |
+| "*" (default)               | Orphan           | Orphan  |
+| contains "Delete"           | Delete (default) | Delete  |
+| contains "Delete"           | Orphan           | Delete  |
+| doesn't contain "Delete"   | Delete (default) | Orphan  |
+| doesn't contain "Delete"   | Orphan           | Orphan  |
+{{< /table >}}
+
 #### Options
-* `managementPolicy: FullControl` - **Default** - Crossplane can create, change
-  and delete the managed resource. 
-* `managementPolicy: ObserveOnly` - Crossplane only imports the details of the
-  external resource, but doesn't make any changes to the managed resource. 
+* `*` - ** Default ** - Crossplane can observe, create, change, delete the
+  external resource and update the managed resource spec with late init
+  parameters.
+* `Observe` - managed resource `status.atProvider` will be updated with the
+  external resource state.
+* `Create` - external resource will be created using the managed resource
+  `spec.initProvider` and `spec.forProvider`.
+* `Update` - external resource will be updated using the managed resource
+  `spec.forProvider`.
+* `Delete` - the external resource will be deleted when the managed resource is
+  deleted.
+* `LateInitialize` - Unprovided spec fields are late initialized to
+  `spec.forProvider` with the default values from the cloud provider. This
+  enables Crossplane to take full control of the external resource, even those
+  values not provided by the user upfront. Read more about [Late Initialization]({{<ref "./managed-resources#late-initialization" >}})
+
+<!-- vale off -->
+### initProvider
+<!-- vale on -->
 
+{{<hint "important" >}}
+The managed resource `initProvider` option is an alpha feature related to 
+[managementPolicies]({{<ref "./managed-resources#managementpolicies" >}}).
+
+Enable the `initProvider` in a provider with `--enable-management-policies`
+in a
+[ControllerConfig]({{<ref "./providers#controller-configuration" >}}).
+{{< /hint >}}
+
+The `spec.initProvider` is a subset of the `spec.forProvider` fields that
+Crossplane merges with `forProvider` to create the external resource, but
+ignores when updating it. This allows users to create a resource with all the
+required fields which may then be controlled externally. For example, a user may
+want to create a `NodeGroup` with the required `desiredSize` field, but then
+let an autoscaler control the field. If the `desiredSize` field is included
+in the`forProvider` field, Crossplane will attempt to update the field, which
+will conflict with the autoscaler.
```

Comment:
Suggested restructure to improve wording and flow and add `{{<hover>}}` support.

```suggestion
{{<hint "important" >}}
The managed resource `initProvider` option is an alpha feature related to 
[managementPolicies]({{<ref "./managed-resources#managementpolicies" >}}).

Enable the `initProvider` in a provider with `--enable-management-policies`
in a
[ControllerConfig]({{<ref "./providers#controller-configuration" >}}).
{{< /hint >}}

The `spec.initProvider` is a subset of the `spec.forProvider` fields that
Crossplane merges with `forProvider` to create the external resource, but
ignores when updating it. This allows users to create a resource with all the
required fields which may then be controlled externally. For example, a user may
want to create a `NodeGroup` with the required `desiredSize` field, but then
let an autoscaler control the field. If the `desiredSize` field is included
in the`forProvider` field, Crossplane will attempt to update the field, which
will conflict with the autoscaler.
```

```
The 
{{<hover label="initProvider" line="7">}}initProvider{{</hover>}} defines 
settings Crossplane applies only when creating a new managed resource.  
Crossplane ignores settings defined in the 
{{<hover label="initProvider" line="7">}}initProvider{{</hover>}} 
field that change after creation.

{{<hint "note" >}}
Settings in `forProvider` are always enforced by Crossplane. Crossplane reverts
any changes to a `forProvider` field.  

Settings in `initProvider` aren't enforced by Crossplane. Crossplane ignores any
changes to a `initProvider` field.
{{</hint >}}

Using `initProvider` is useful for setting initial values that a Provider may
automatically change, like an auto scaling group.

For example, creating a 
{{<hover label="initProvider" line="2">}}NodeGroup{{</hover>}} 
with an initial 
{{<hover label="initProvider" line="9">}}desiredSize{{</hover>}}.  
Crossplane doesn't change the 
{{<hover label="initProvider" line="9">}}desiredSize{{</hover>}}
setting back when the Provider auto scales the Node Group.

{{< hint "tip" >}}
Crossplane recommends configuring
{{<hover label="initProvider" line="6">}}managementPolicies{{</hover>}} without
`LateInitialize` to avoid conflicts with `initProvider` settings.
{{< /hint >}}
```

Reply:
Applied, just added a slight change to

```
Settings in `forProvider` are always enforced by Crossplane. Crossplane reverts
any changes to a `forProvider` field.  

Settings in `initProvider` aren't enforced by Crossplane. Crossplane ignores any
changes to a `initProvider` field.
```
to 
```
Settings in `forProvider` are always enforced by Crossplane. Crossplane reverts
any changes to a `forProvider` field in the external resource.

Settings in `initProvider` aren't enforced by Crossplane. Crossplane ignores any
changes to a `initProvider` field in the external resource.
```

To make a clear distinction in making changes in managed its external resource